functions.yml

```diff
@@ -366,12 +383,14 @@ quality_control:
           [Row(stats=[Row(sampleId='NA12878', mean=2.0, stdDev=1.0, min=1.0, max=3.0)])]
       args:
         - name: genotypes
-          doc: The array of genotype structs
+          doc: An array of genotype structs with a ``conditionalQuality`` field
```

Comment:
We should be consistent with whether we preface the field name with `a` or not

Reply:
Fixed